docs/targets/zendesk.md

```diff
@@ -0,0 +1,97 @@
+# Event Target for Zendesk
+
+This event target receives [CloudEvents][ce] and uses the Zendesk API to either create a new ticket or tag an existing one.
+
+## Prerequisites
+
+A Zendesk API token is required to utilize this Target. The steps to obtain a token are outlined in the [Zendesk API Docs](https://support.zendesk.com/hc/en-us/articles/226022787-Generating-a-new-API-token-)
+
+## Deploying an Instance of the Target
+
+Open the Bridge creation screen and add a Target of type `Zendesk`.
```

Comment:
@cab105 following your logic, this sentence is wrong because it uses the 2nd person. Mixing 3rd and 2ns persons should really be avoided.

Reply:
You are correct in that the mixing of personas should be avoided, and I take it you are referencing the implicit `you` in this case?

Reply:
Yes, initiative would fall in the 2nd person right? Which means, if we rephrased other parts of the docs specifically to avoid the 2nd person, we should also get rid of all imperatives, correct?

Reply:
honestly, imperative and/or 2nd person is just fine.  How can we drive this to a merge so that we can also move on to the other 3 PRs and merge the frontend forms :)

Reply:
?

Reply:
It's approved already isn't it?

Reply:
If we wanted to scrub the implicit `you` in this case, then rephrasing it as "Begin by opening the bridge..."

However, given some of the comments left in the other PRs regarding the blending or allowing 2nd person I think we've established that we're okay with it. What I'd ask for is that we're consistent throughout.

Reply:
Yeah, same here, consistent is the only thing I care about.

Reply:
ok, so let's use the 2nd person or imperative sentence consistenly. Do we agree ?

Reply:
Think the grammar police would agree.